### DISCUSSION TITLE
Problem exporting JSON

### DISCUSSION THREAD

**Discussion by Spyfut** [Xzandro/sw-exporter#405] [Q&A]
Hello, 

First of all thanks for the great job you did on this app ! :) 

I have a problem when i do the setup to get the json file, it does never export a json file.
All the process is respected (i use the last version of nox to emulate android), wifi ip, certification etc, could it be a problem with Hive logged via Facebook ? 
I only see this difference for me, i loggin to hive with my facebook account. 

Have you a similar problem these days with the export ? 

Thank you very much ! :)

**Comment by Xzandro** [comment]
Sounds like you never enabled HTTPS mode. It should not matter if you login via facebook or hive.

**Reply by Spyfut** [reply]
Hey ! :) I found a solution to my problem, HTTPS mode was okay, but it worked when i unchecked the boxes for swarfarm options

**Comment by SuiSidee** [comment]
maybe in 0.0.45 this exporter will finally work :(

**Reply by Xzandro** [reply]
It already does work within its limitations. This will not change in the future.

**Reply by SuiSidee** [reply]
I did everything and still can't get a JSON file, so I guess it's not obtainable anymore.

**Reply by Xzandro** [reply]
It is. Stop spreading false information.

**Reply by goldeen7** [reply]
Hello Xzandro,
Firstly, great job with the Optimizer! It lets people easily manage runes and units in Summoners!
I can't get it to work. :( The json file would not export.
I used Optimizer and SWOP in the past and it worked great. Now it doesn't. Last I used it was May 2020. :) I have Android 6, so it should be good.
So... I installed the cert (got the current one from the Exporter), I conected to WiFi, I inserted the Proxy digits from the Exporter. I enabled HTTPS in the Exporter. (just to be save I disabled the SWAFARM options). I clicked Start Proxy, then started Summoners. The screen on my mobile is black for a good several seconds before Summoners starts. I do not see the usual "the json file exported successfully" in the Exporter. I wait forever then I get the timeout.
What do I need to fix to make it work? Please help!

**Reply by SuiSidee** [reply]
Telling the truth is spreading false information. Incredible.

**Comment by Frankswolv** [comment]
Hi,

is there an option without rooting or use an very old device (unfortunately I don't have one that work and I won't use bluestack or other semilar solutions) to get the json file? I try the above tips (firewall shutdown, https "on" in the Summoners War Exporter, getting cert but its impossible to install on my device ...) but I get only the information that I have connection problems.....

My phone is:

Red Magic 6 Pro with Android 11...

Is there an solution do get the json file?????


Have a nice day

Frank

**Reply by Xzandro** [reply]
Not really. You are pretty much stuck with an emulator, Android 6 or lower or iOS.

**Comment by antoine-mait** [comment]
Hi , 
Just to tell you that the version 0.0.64 doesnt work to export the JSON for MuMu player 12 ( the one i use ) but the version 0.0.62 works perfectly. I've tried with and without certification , install reinstall completly with deleting the user data but nothing do :/ Sorry i can't help more pointing where the issue is ! Thank you for it tho, it always worked perfectly before
